cms/templates/cms/home_page.html

```diff
@@ -66,15 +66,16 @@
         <div class="col-md-12 ">
           <div class="sub-banner-section">
             <div class="sub-banner-text">
-            <h3>What is MicroMasters Credential?</h3>
+            <h3>What is the MicroMasters Credential?</h3>
```

Comment:
Do we want to make this change? Are we expecting more changes to this copy shortly?